### PR TITLE
IZE-391 IZE-392 describe new structures for new file configuration design

### PR DESCRIPTION
**Pull request by psihachina**
## Changelog:
- rename projects folder to apps folder in example ecs-apps-monorepo
- describe new structures for new file configuration design
- rename `projects_path` parameter to `apps_path`

#### apps_path
[asciicast](https://asciinema.org/a/SclUsh3KbqoIlWSAGbWx4Z5L3)

#### new structures
[asciicast](https://asciinema.org/a/IXWMcvG8Y6a1Uf54WrwVRtPb4)

### CODE REVIEW

**Review by AutomationD — CHANGES_REQUESTED** [review]
Looks good, I had a quick note about default path and questions about schema fallback

File: internal/schema/ize-spec.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
         "prefer_runtime": {
             "type": "string"
         },
-        "projects_path": {
+        "apps_path": {
             "type": "string"
```
[the remark this answers is not in the document]
Comment (psihachina) [reply internal/schema/ize-spec.json:39]: must work, i used only viper and it must supported any file format

File: internal/schema/ize-spec.json [hunk 2]
```diff
@@ -51,8 +51,44 @@
...
         },
         "app": {
+            "id": "#/properties/app",
             "type": "object",
-            "properties": {}
+            "patternProperties": {
```
[the remark this answers is not in the document]
Comment (psihachina) [reply internal/schema/ize-spec.json:56]: I tried to describe the old structure in a schema, but we wanted to convert it into new structures after reading the old structure. The next step is to write a "converter"

File: internal/config/config.go [hunk 3]
```diff
@@ -180,7 +179,7 @@ func InitConfig() {
 	}
 
 	viper.SetDefault("ROOT_DIR", cwd)
-	viper.SetDefault("PROJECTS_PATH", filepath.Join(cwd, "projects"))
+	viper.SetDefault("APPS_PATH", filepath.Join(cwd, "apps"))
```
[the remark this answers is not in the document]
Comment (psihachina) [reply internal/config/config.go]: @AutomationD 
[asciicast](https://asciinema.org/a/2lj1J0DpeOOtJR42lZQw6DrBK)

Is that okay?

**Review by AutomationD — APPROVED** [review]

**Pull request merged**